src/wrap_cl_part_2.cpp

```diff
@@ -32,6 +32,11 @@
 
 
 
+void dummy_fn() {}
+PyType_Slot slots[] = {
```

Comment:
I think I'd prefer to just move the implementation of the constructors to C++, instead of this slots hackery.